src/store/buy-with-credit-card/utils.ts

```diff
@@ -86,13 +119,29 @@ export const mapUtorgProviderCurrencies = (currencies: UtorgCurrencyInfo[]) => (
     }))
 });
 
-export const mapAliceBobProviderCurrencies = ({ minAmount, maxAmount }: AliceBobPairInfoResponse) => ({
-  fiat: [
-    {
-      ...aliceBobHryvnia,
-      minAmount: minAmount,
-      maxAmount: maxAmount
+export const mapAliceBobProviderCurrencies = ({ pairsInfo }: AliceBobPairsInfoResponse) => ({
+  fiat: pairsInfo.map(pair => {
+    console.log(pair);
+    const [minAmountString, code] = pair.minamount.split(' ');
+    const minAmount = Number(minAmountString);
+    const maxAmount = Number(pair.maxamount.split(' ')[0]);
+
+    if (isDefined(knownAliceBobFiatCurrencies[code])) {
+      return {
+        ...knownAliceBobFiatCurrencies[code],
+        minAmount,
+        maxAmount
+      };
     }
-  ],
+
+    return {
+      name: getCurrencyNameByCode(code),
+      code,
+      icon: `https://static.moonpay.com/widget/currencies/${code.toLowerCase()}.svg`,
+      precision: 2,
```

Comment:
Please check the order creation flow with JPY - it has no decimals. Just resolve this conversation if it works well

Reply:
it works fine